scripts/Inelastic/Direct/AbsorptionShapes.py

```diff
@@ -6,6 +6,7 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 
 from __future__ import (absolute_import, division, print_function)
```

Comment:
Is it worth adding `unicode_literals` to the future import list here?

Reply:
I've added `unicode_literals` the imports for all the direct reduction scripts, and I've also changed the target to `release-next`. I think this means that it will get merge into both `release-next` and `master`?

Reply:
Thanks. Yep. We periodically merge `release-next` to `master` so that'll take care of itself.